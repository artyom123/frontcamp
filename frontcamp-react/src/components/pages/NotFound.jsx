import React from 'react';
import {
    makeStyles,
    Container,
    Grid,
    Typography,
    Link,
} from '@material-ui/core';

import Footer from '../footer/Footer';

const NotFound = () => (
    <Container>
        <Grid>
            <Typography variant="h1" component="h2" gutterBottom>
                Page not found
            </Typography>
            <Link href="/">
                Go to main page
            </Link>
        </Grid>
        <Footer/>
    </Container>
);

export default NotFound;
