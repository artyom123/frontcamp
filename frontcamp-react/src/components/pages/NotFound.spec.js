import React from 'react'
import { shallow } from 'enzyme';
import NotFound from './NotFound';

describe('NotFound', () => {
    const notFound = shallow(<NotFound />);

    it('render initial', () => {
        expect(notFound).toMatchSnapshot();
    });
});