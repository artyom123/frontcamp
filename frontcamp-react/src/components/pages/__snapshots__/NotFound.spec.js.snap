// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NotFound render initial 1`] = `ShallowWrapper {}`;
